### PR TITLE
Minimal documentation on howto create and configure a runtime-api project with SDK 6.

### PR DESCRIPTION
**Pull request by bguedas**

### CODE REVIEW

**Review by denisbedin — CHANGES_REQUESTED** [review]
@bguedas You can also add a section in https://docs.microej.com/en/latest/SDK6UserGuide/moduleNatures.html to document the different tasks that is added by the plugin.

**Comment by tdelhomenie-microej** [comment]
@bguedas we should also have a SDK6 version of the page https://docs.microej.com/en/latest/KernelDeveloperGuide/runtimeEnvironment.html (with the "extend" section).

File: KernelDeveloperGuide/runtimeEnvironment.rst [hunk 1]
```diff
@@ -30,14 +30,28 @@ The following figure shows the overall build flow:
 Create a new Runtime Environment Module
 ---------------------------------------
 
-A Runtime Environment :ref:`module project <mmm_module_skeleton>` is created with the ``runtime-api`` skeleton.
+.. tabs::
+
+   .. tab:: Gradle (build.gradle.kts)
```
Comment (tdelhomenie-microej) [inline KernelDeveloperGuide/runtimeEnvironment.rst:35]: We use `SDK 6` and `SDK 5` in the tabs, we should be consistent IMO.

File: KernelDeveloperGuide/runtimeEnvironment.rst [hunk 2]
```diff
@@ -30,14 +30,28 @@ The following figure shows the overall build flow:
 Create a new Runtime Environment Module
 ---------------------------------------
 
-A Runtime Environment :ref:`module project <mmm_module_skeleton>` is created with the ``runtime-api`` skeleton.
+.. tabs::
```
Comment (tdelhomenie-microej) [inline KernelDeveloperGuide/runtimeEnvironment.rst:33]: Shouldn't we create a dedicated section in SDK 5 and SDK 6 documentations, as we did for Applications and Libs?

**Review by tdelhomenie-microej — COMMENTED** [review]
@bguedas the build fails because of this error:
```
/home/docs/checkouts/readthedocs.org/user_builds/microej-docs/checkouts/feature-sdk6_runtime_api/KernelDeveloperGuide/runtimeEnvironment.rst:36: WARNING: undefined label: '_sdk_6_create_project:'
```

File: KernelDeveloperGuide/runtimeEnvironment.rst (hunk 1 above)
Comment (bguedas) [reply KernelDeveloperGuide/runtimeEnvironment.rst:35]: Actually, I reused the tabs from https://docs.microej.com/en/latest/ApplicationDeveloperGuide/runtime.html for consistency :) But Ok, i'll change the latter to SDK 5/SDK 6

Comment (tdelhomenie-microej) [reply KernelDeveloperGuide/runtimeEnvironment.rst:35]: Ah yes, right :)

File: SDK6UserGuide/moduleNatures.rst [hunk 3]
```diff
@@ -111,6 +111,29 @@ This plugin adds the following tasks to your project:
...
+-----------
+
+**Plugin Name**: ``com.microej.gradle.runtime-api``
+
+**Tasks**:
+
+This plugin adds the following tasks to your project:
+
+- tasks of the `Gradle Java plugin <https://docs.gradle.org/current/userguide/java_plugin.html>`__
+- :ref:`sdk6_module_natures.tasks.checkModule`
```
Comment (denisbedin) [inline SDK6UserGuide/moduleNatures.rst:126]: In the `checkModule` task section, can you add the runtime-api plugin to the list of plugins that use the task?

**Review by denisbedin — CHANGES_REQUESTED** [review]

File: SDK6UserGuide/moduleNatures.rst (hunk 3 above)
Comment (bguedas) [reply SDK6UserGuide/moduleNatures.rst:126]: done

File: KernelDeveloperGuide/runtimeEnvironment.rst (hunk 1 above)
Comment (bguedas) [reply KernelDeveloperGuide/runtimeEnvironment.rst]: Changed

**Review by denisbedin — APPROVED** [review]

**Review by tdelhomenie-microej — APPROVED** [review]

**Pull request merged**